bundler/spec/fixtures/projects/bundler1/tool_versions_file/.gitignore

```diff
@@ -0,0 +1 @@
+!.tool-versions
```

Comment:
`.tool-versions` is ignored at the root of the project, and so we need to negate it here to add this file to the fixture project.